src/store/users/slice.js

```diff
@@ -0,0 +1,39 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import API from '../../api/api';
+
+export const getUsers = createAsyncThunk(
+  'users/getUsers',
+  async (_, { rejectWithValue }) => {
+    try {
+      const response = await API.get(`/users`);
+      return response.data;
+    } catch (err) {
+      return rejectWithValue(err.response.data);
+    }
+  }
+);
+
+const usersSlice = createSlice({
+  name: 'users',
+  initialState: {
+    users: [],
+    status: 'idle',
+    error: null,
+  },
+  reducers: {},
+  extraReducers: {
+    [getUsers.pending]: (state) => {
+      state.status = 'loading';
```

Comment:
you can add a new line under this one:

state.users = []; to initialize the state of the users array.